src/commands/pickFuncProcess.ts

```diff
@@ -122,3 +132,7 @@ interface IProcessTreeNode {
 interface IWindowsProcessTree {
     getProcessTree(rootPid: number, callback: (tree: IProcessTreeNode | undefined) => void): void;
 }
+
+async function delay(ms: number): Promise<void> {
```

Comment:
Delay is a pretty generic function.  Should we move it to somewhere more common?

Reply:
Yeah it's pretty generic (I mostly copied it from a util function Stephen had).

I can go either way on this one. On one hand, I don't like how the 'utils' folder is just a pile of random functions, so I try not to put stuff in there until we're actually using it multiple times. On the other hand, I feel like that could lead to more duplicated work because the next person is less likely to find this function since it's not in utils. Thoughts? In any case, I think I'll just leave as is for this PR.

Reply:
Always could wait until someone accidentally duplicates it and move it then.  I agree it's less discoverable, but I'm sure one of us will remember come PR time... 😅

Reply:
I wouldn't mind seeing it in ui.  Maybe I'll add it when I'm doing other stuff there.